src/apiClient.js

```diff
@@ -2349,7 +2349,8 @@ class ApiClient {
             url,
             data: {
                 NewPw: newPassword
-            }
+            },
+            contentType: 'application/json'
```

Comment:
Thinking about it, wouldn't adding the field to `ajax()` directly fix it everywhere?